### PR TITLE
fix: Update logging library

### PR DESCRIPTION
**Pull request by averikitsch**
## Description

Fixes #5697

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: run/logging-manual/e2e_test.py [hunk 1]
```diff
@@ -188,12 +188,14 @@ def test_end_to_end(service_url_auth_token, deployed_service):
...
 
     # Retry a maximum number of 10 times to find results in stackdriver
+    found = False
     for x in range(10):
-        iterator = client.list_log_entries(resource_names, filter_=filters)
+        iterator = client.list_log_entries({"resource_names": resource_names, "filter": filters})
         for entry in iterator:
+            found = True
             # If there are any results, exit loop
             break
```
Comment (dandhlee) [inline run/logging-manual/e2e_test.py:197]: If a result is found, do we want to break out of the loop any earlier? Or even if the result is found do we want to run all 10 times?

Comment (averikitsch) [reply run/logging-manual/e2e_test.py:197]: Yes! good catch

File: run/logging-manual/e2e_test.py [hunk 2]
```diff
@@ -188,12 +188,17 @@ def test_end_to_end(service_url_auth_token, deployed_service):
...
+        # When message found, exit loop
+        if found is True:
```
Comment (dandhlee) [inline run/logging-manual/e2e_test.py:199]: nit: `if found:` should be good enough.

**Review by dandhlee — APPROVED** [review]
LGTM, one nit below!

**Review by busunkim96 — APPROVED** [review]
@daniel-sanche Could you check the logging library is being used as you expect? Thank you!

**Pull request merged**